addons/breathing/functions/fnc_handleBreathing.sqf

```diff
@@ -202,3 +202,21 @@ if (!local _unit) then {
         };
     };
 }, 3, [_unit]] call CBA_fnc_addPerFrameHandler;
+
+
+
+[{
+    params ["_args", "_idPFH"];
+    _args params ["_unit"];
+
+    private _status = _unit getVariable [QGVAR(airwayStatus), 100];
+
+    if(!(alive _unit) || _status > 100 || _unit getVariable ["ACE_isUnconscious", false]) exitwith {
```

Comment:
Use macros.

Instead of "_unit getVariable ["ACE_isUnconscious", false]"
use IS_UNCONSCIOUS(_unit)